src/slurm_plugin/slurm_resources.py

```diff
@@ -166,11 +168,7 @@ def is_drained(self):
 
         drained(sinfo) is equivalent to IDLE+DRAIN(scontrol) or DOWN+DRAIN(scontrol)
         """
-        return (
-            self._is_drain()
-            and not self.is_power_down()
-            and (self.SLURM_SCONTROL_IDLE_STATE in self.states or self.is_down())
-        )
+        return self._is_drain() and (self.SLURM_SCONTROL_IDLE_STATE in self.states or self.is_down())
```

Comment:
is_drained() is not used in the is_rebooting logic now, right? You are using is_drain() in the is_rebootong logic. Shall we recover this function to check also `not self.is_power_down()`?

Reply:
No, I really want to check just the `DRAIN` flag (so `_is_drain()`) within the logic `is_rebooting()`. The case for `is_down()` is treated in the `elif` below, and has a slightly different logic.

The previous usage of `is_drained()` was coming from the structure of `is_healthy_state()`, but it's better to use `_is_drain()`.